guide/14-deep-learning/panoptic_segmentation_with_maxdeeplab.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>fix formatting issue in the last two words</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
That is done deliberately to highlight the name of the model.